src/Modules/Reputation/ReputationModule.sol

```diff
@@ -0,0 +1,323 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.17;
+
+import { ReputationTokenInterface } from "./interfaces/ReputationTokenInterface.sol";
+import { ReputationModuleInterface } from "./interfaces/ReputationModuleInterface.sol";
+import { Clones } from "@openzeppelin/contracts/proxy/Clones.sol";
+
+contract ReputationModule is ReputationModuleInterface {
+    using Clones for address;
+
+    address public network;
+
+    mapping(address => ReputationMarketConfig) public laborMarketRepConfig;
+
+    event ReputationTokenCreated (
```

Comment:
Looking for opinion: Why are these not utilizing the same event methodology? Why use data that can't be recovered for one and a struct for the other?